app/routes/circulars/circulars.lib.ts

```diff
@@ -158,6 +168,11 @@ export const validSubjectKeywords = [
   'ZTF',
 ]
 
+export const erroneousSubmissionKeywords = [
```

Comment:
Rename to make it clear that these are not partial matches, these are whole subjects.

```suggestion
export const erroneousSubmissionSubjects = [
```

Reply:
Good idea, I'll have that in the next commit